src/components/notebook-utils.js

```diff
@@ -91,7 +91,7 @@ export const toolExtensionDisplay = {
 }
 
 export const getPatternFromTool = toolLabel => Utils.switchCase(toolLabel,
-  [tools.RStudio.label, () => '.*\\.R(md)'],
+  [tools.RStudio.label, () => '.*(.R|.Rmd)$'],
```

Comment:
Should the notebook name contain at least one character?

```suggestion
  [tools.RStudio.label, () => '.+(.R|.Rmd)$'],
```

Reply:
`+` is not better than `*`. Consider an case of one letter notebook name. Regex with `+` won't match `t.R`. 
You may want to consider an edge case with file extension occurs more than once. Such as `hello.R.R` or `helo.Rmd.Rmd` or `hello.Rmd.R`.
Consider using this regex or come up with a better version: `.+[^(.R|.Rmd)](.R|.Rmd)$`

Reply:
@aweng98 it will match things like `t.R`...regarding the repeating `.R.R`, I'm leaning towards keeping things simple for now until we have a case to make further decisions. Users can create arbitrary file names in terminal, and we can't really prevent them from doing so. This regex here is just determining what files will be synced to workspace bucket.

Reply:
Totally. Thanks.